docs/check.md

```diff
@@ -0,0 +1,439 @@
+# Preflight Check
+
+:warning: :construction:
+**This page is about the deprecated check command**
+:warning: :construction:
+
+The Preflight _check_ tool also uses _data gatherers_ to collect required data,
+but unlike the _agent_ it evaluates this data and produces a report locally.
+
+**This functionality is now deprecated in favour of the _agent_.** Previous
+versions of the _check_ tool can still be downloaded and used, however it is no
+longer being maintained and will be removed from this repository.
+
+## Packages
```

Comment:
The packages are still in use so I think this section should either stay in `README.md` or move to `docs/packages.md`

Reply:
But they're not in use by users of the agent - and will likely go before they come back from the public repo.

Reply:
Ah yeah! I might PR this paragraph back into the private repo's `README.md` then.

Reply:
Actually I think it would still be good to briefly mention packages. Maybe just:

```
## Packages

Policies for cluster configuration are encoded into *Preflight packages*. 
Each package focuses on a different aspect of the cluster, for example the
`gke` package provides rules for the configuration of a GKE cluster.

Preflight packages are implemented using
[Open Policy Agent](https://www.openpolicyagent.org) with evaluation
taking place in the SaaS backend.
```

Reply:
Yeah, good idea, added in: 835d92fd9e963624e9bbe8ec67839624424578d0